services/QuillLMS/client/app/bundles/Diagnostic/modules/translation/languagePageInfo.js

```diff
@@ -1,100 +1,14 @@
-export const languages = ['english', 'arabic', 'chinese', 'french', 'hindi', 'spanish', 'vietnamese'];
-export const languagesV2 = ['english', 'chinese', 'hindi', 'spanish', 'french', 'arabic', 'russian', 'portuguese', 'urdu', 'german', 'japanese', 'korean', 'vietnamese', 'thai', 'ukrainian', 'tagalog', 'dari'];
+export const languages = ['english', 'chinese', 'hindi', 'spanish', 'french', 'arabic', 'russian', 'portuguese', 'urdu', 'german', 'japanese', 'korean', 'vietnamese', 'thai', 'ukrainian', 'tagalog', 'dari'];
 export const rightToLeftLanguages = ['arabic', 'urdu', 'dari'];
```

Comment:
we should probably constantize the individual words since we do go on to use them as keys throughout this file